### ISSUE TITLE
Account Search - Listing Display

### ISSUE THREAD

**Issue by samuraiseoul** [samuraiseoul/test#26]
Currently showing separate line item for each contact under the same account.  Would prefer to only show one listing and change results from displaying contact name and related info to the following:

Account Name (not Customer Name), Location, City, State, Phone, Account Type

Also, while changing this, can you get rid of the View Details column and just make the Account Name the link?

**Comment by samuraiseoul** [comment]
[[image:ad4trGpaCr4QJXacwqjQWU]]
Account Search Results View

**Comment by samuraiseoul** [comment]
Michael, I requested above that the data be changed to:

Account Name (not Customer Name), Location, City, State, Phone, Account Type

instead of:

Customer Name, Contact Name, Phone, Email, View Details.  We need to make this change, but then also still have an alternate solution for name search.  I know what I want but probably need to discuss this again.

Thanks for changing the company search by making that the link.  So much easier and better.

**Comment by samuraiseoul** [comment]
Sorry just totally didn't read this one.

**Comment by samuraiseoul** [comment]
You stated the contact search was fixed, but when searching on a contact name like \"saper\", it's still pulling up the company name with the first name only of the first listing for that company.  It's not showing Karen's contact information?

**Comment by samuraiseoul** [comment]
Can you fix this one this evening or can we further discuss?

**Comment by samuraiseoul** [comment]
The problem with \"Saper\" search noted above still exists.  

Also regarding the Account search results list, can you please add City and State immediately after the Customer Name column?  If you have to delete a column due to space restrictions, get rid of Email.

Also, please change \"Customer Name\" to \"Account Name\" as they aren't all customers.

**Comment by samuraiseoul** [comment]
After getting the sales team to start using this, would like to change the headers to the following:

Account Name (instead of Customer Name), City, State, Contact Name, Phone, Account Owner

We would also like to be able to sort this list by any column by clicking on the column header.  The default should be alphabetical order by company name.  If sorting these displays list is a significantly bigger project, then we need to wait on that.

Also, if there are multiple pages in the search results list.  Can you either move the option to go the more pages to the bottom right corner or, better yet, add the option at the bottom of the page (keeping it as well at the top).

**Comment by samuraiseoul** [comment]
(In [[r:8ed421e936ea64bdea9d0956e974306cc0f59bed|lifestyle-crm:8ed421e936ea64bdea9d0956e974306cc0f59bed]]) change Customer name to account name see #26

Committed to: lifestyle-crm:bugfixes_2012_12

**Comment by samuraiseoul** [comment]
splitting paging and sorting to new ticket

**Comment by samuraiseoul** [comment]
(In [[r:a1293ef632d572d0998bfed14c12a562a4b73a8f|lifestyle-crm:a1293ef632d572d0998bfed14c12a562a4b73a8f]]) add city and state to account list complete #26

Committed to: lifestyle-crm:bugfixes_2013_01

**Comment by samuraiseoul** [comment]
Fixed
